timemachine/potentials/summed.py

```diff
@@ -41,3 +38,28 @@ def summed_potential(
     split_indices = np.cumsum(sizes)
     paramss = [ps.reshape(shape) for ps, shape in zip(np.split(params, split_indices[:-1]), shapes)]
     return sum(U_fn(conf, ps, box) for U_fn, ps in zip(U_fns, paramss))
+
+
+def fanout_summed_potential(
+    conf: Conf,
+    params: Params,
+    box: Optional[Box],
+    U_fns: Sequence[PotentialFxn],
+):
+    """Reference implementation of the custom_ops FanoutSummedPotential.
+
+    Parameters
+    ----------
+    conf: array (N, 3)
+        conformation
+
+    params: array (P,)
+        flattened array of parameters shared by each potential term
+
+    box: array (3, 3)
+        periodic box
+
+    U_fns: list of functions with signature (conf, params, box) -> energy
+        potential terms
+    """
+    return sum(U_fn(conf, ps, box) for U_fn, ps in zip(U_fns, jnp.array(params)))
```

Comment:
I'm not aware of any significant difference. I guess I was thinking that since this is just summing floats (vs. arrays), it would be simpler to use the stdlib function. On second thought though, I wonder if there might be a difference when we jax-transform the output. Might change to `jnp.sum(jnp.array(...))` for better consistency with existing code.

Reply:
Changed to `jnp.sum(jnp.array(...))` for consistency in https://github.com/proteneer/timemachine/pull/995/commits/97526fafcbce8ef1fc4e18464024ba185cb8232a